apps/devhub/tests/test_views.py

```diff
@@ -1190,6 +1190,28 @@ def test_create_new_credentials(self):
         mock_creator.assert_called_with(self.user)
         self.assertRedirects(response, self.url)
 
+    def test_retreive_credentials(self):
+        patch = mock.patch('api.models.APIKey.get_jwt_key')
+        with patch as mock_creator:
+            mock_creator.return_value = APIKey(
+                key='abc123',
+                secret='supersekret',
+            )
+            response = self.client.get(reverse('devhub.api_creds'))
+
+        assert response.status_code == 200
+        response_json = json.loads(response.content)
+        assert response_json['key'] == 'abc123'
+        assert response_json['secret'] == 'supersekret'
+
+    def test_retreive_credentials_404(self):
+        patch = mock.patch('api.models.APIKey.get_jwt_key')
+        with patch as mock_creator:
```

Comment:
same thing here applies about not using a mock. This test will be easier since you don't have to create an APIKey object to receive a 404.